src/rules/sort-collections.ts

```diff
@@ -29,14 +29,38 @@ export const rule = createRule<Options>({
 					toSort.includes(key.value)
 				) {
 					const currentOrder = collection.properties;
-					const desiredOrder = currentOrder
-						.slice()
-						.sort((a, b) =>
-							(a.key as AST.JSONStringLiteral).value >
-							(b.key as AST.JSONStringLiteral).value
-								? 1
-								: -1,
-						);
+					const desiredOrder = currentOrder.slice().sort((a, b) => {
+						let aKey = (a.key as AST.JSONStringLiteral).value;
+						let bKey = (b.key as AST.JSONStringLiteral).value;
```

Comment:
Not sure if you prefer this or
```suggestion
						let { value: aKey } = a.key as AST.JSONStringLiteral;
						let { value: bKey } = b.key as AST.JSONStringLiteral;
```

Reply:
Unless a lint rule is yelling at us, I think it's fine as you have it. It's the existing code style and fewer characters.